packages/backend/src/build-disk-image.ts

```diff
@@ -176,13 +176,14 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
 
           // Step 3.1 Since we have started the container, we can now go get the logs
           await logContainer(build.engineId, containerId, progress, data => {
-            async () => {
+            // update the log file asyncronously
+            void (async () => {
               try {
```

Comment:
AFAIK you should not use the void keyword there
Just call the function without await but catch error

Reply:
You're right. Removed void and rebased on main.